lib/shared/config-validator.js

```diff
@@ -148,7 +149,10 @@ export default class ConfigValidator {
         const validateRule = ruleValidators.get(rule);
 
         if (validateRule) {
-            validateRule(localOptions);
+            const mergedOptions = deepMergeArrays(rule.meta?.defaultOptions, localOptions);
```

Comment:
We can't use `?` in this package because it's also used by ESLint v8, which supports older Node.js versions that don't have support for this syntax.

Reply:
The minimum Node.js version currently supported is [18.18.0](https://github.com/eslint/eslintrc/blob/v3.1.0/package.json#L80), which does recognize the optional chain operator syntax `?.`.